designer/server/plugins/routes/api.ts

```diff
@@ -71,6 +71,13 @@ export const putFormWithId: ServerRoute = {
         return h.response({ ok: true }).code(204);
       } catch (err) {
         console.error("Designer Server PUT /api/{id}/data error:", err);
+        const errorSummary = {
+          id: id,
+          payload: request.payload,
+          errorMessage: err.message,
+          error: err,
```

Comment:
If the whole `err` object is going into `errorSummary.error` do we also need `errorSummary.errorMessage`? Isn't that going to store the same thing as `errorSummary.error.message`?

Reply:
I noticed some issues with Error object not being serialized to json , so as fallback I used errorMessage
as per some stackoverflow  -  error.stack can be used instead of trying to serialize whole of error  , but I did not give it a try

Reply:
changed it to error stack